examples/js/archive.js

```diff
@@ -164,10 +172,13 @@ restoreWelcomeModal.addEventListener('click', (event) => {
 
 mapToggle.addEventListener('click', (event) => {
   new bootstrap.Offcanvas(sidebar).show();
+  sidebarOpen = true;
 });
 
 tileMap.addEventListener('click', (event) => {
-  bootstrap.Offcanvas.getInstance(sidebar).hide();
+  if (sidebarOpen) {
+   bootstrap.Offcanvas.getInstance(sidebar).hide();
```

Comment:
Just looked this up and I do see Offcanvas has a "toggle" method? Could save a few lines of code maybe!

https://getbootstrap.com/docs/5.0/components/offcanvas/#methods

Reply:
Okay...thanks!

Reply:
The "toggle" method works but then it generates an unintended effect when applied on the tile layer event listener. Clicking on tile layer should only close the sidebar if open and nothing more but using the "toggle" method means, the sidebar is switched on and off whenever the tile layer receives a click event. For this reason, I think we don't need the toggle method.